core/new-gui/src/app/workspace/component/code-editor-dialog/code-editor-dialog.component.ts

```diff
@@ -56,9 +61,47 @@ export class CodeEditorDialogComponent implements AfterViewInit, SafeStyle, OnDe
 
   ngOnDestroy(): void {
     this.workflowActionService.getTexeraGraph().updateSharedModelAwareness("editingCode", false);
+    if (this.socket) {
```

Comment:
better be explicit with this condition. in javascript a lot of edge case can be evaluated as false. try to use helper function `isDefined` or `!==undefined`?